public/javascripts/SVLabel/src/SVLabel/Storage.js

```diff
@@ -39,6 +39,10 @@ function TemporaryStorage(JSON, params) {
             set("completedOnboarding", null);
         }
 
+        if (!get("completedMissionAnonymously")){
```

Comment:
Wouldn't it be correct to update this to `completedFirstMission`? Because we also want to show the first mission to a signed in user if they haven't done any missions.